android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMManager.java

```diff
@@ -1757,6 +1763,20 @@ public void run() {
             }
           }
 
+          // Perform right-deletions
```

Comment:
This part is still TODO, right?

Reply:
Actually, that part's not TODO - the system keyboard's version was really easy code.  The in-app one though... that's a bit daunting, so that one is.